README.md

```diff
@@ -83,14 +83,11 @@ dependencies {
 
 The `Trusona` class is the main class you will interact with to talk to the Trusona APIs. It can be created with the `token` and `secret` provided by [Trusona](#server-sdk-api-credentials).
 
-*NOTE:* The `token` and `secret` should not be shared with anyone. They are how you authenticate to the Trusona services, and you should not check them into source control.
-
 ```java
 Trusona trusona = new Trusona(token, secret);
 ```
 
-You'll also want to make sure the `token` and `secret` values aren't checked in to your project.
```

Comment:
This is basically repeated in the note above. 

I kept the first note and moved it below the example. There is another note about this in the section above but that one didn't bother me as much as this being right after each other.